docs/dasharo-menu-docs/overview.md

```diff
@@ -58,6 +58,37 @@ board variant.
   used for setup application. Lists all bootable options and allows one to
   override the boot path.
 
+### Hotkeys
+
+In the standard EDK2 BIOS, the functionality of the F9 and F10 hotkeys is
+limited to certain menus and applies only to the settings within the current
+menu or entry. In Dasharo, these hotkeys are present and effective globally.
+
+#### F9: Reset to Defaults
+
+* The `F9` hotkey is a **global reset option**.
+* Pressing `F9` will reset all BIOS settings across all menus to their default
+values.
+* This action is immediate and applies to all configurable options within the
```

Comment:
We have one exception, the secure boot keys will not be reset to defaults.

Reply:
Change applied [here](https://github.com/Dasharo/docs/pull/709/files#diff-256c66f72451358d31ff2d64488936acff63f53b0b36cbb8e1f0be9b9d5406d0R67)